services/document-fetcher.js

```diff
@@ -59,16 +59,30 @@ export function documentFetcher(
     getPreviewUrl: (previewToken) => {
       return prismic.api.previewSession(previewToken, LinkResolver, '/')
     },
-    getSimplePageByUid: async (uid) => {
-      const document = await prismic.api.query(
-        prismic.predicates.at('my.simple_page.uid', uid),
-        { lang }
-      )
-      return document.results[0]
+    getPageByUid: async (uid) => {
+      const simplePage = await getSimplePageByUid(uid)
+      const slicePage = await getSlicesPageByUid(uid)
+      return simplePage || slicePage
     },
   }
 
   function getSingle(type) {
     return prismic.api.getSingle(type, { lang })
   }
+
+  async function getSimplePageByUid(uid) {
+    const document = await prismic.api.query(
+      prismic.predicates.at('my.simple_page.uid', uid),
```

Comment:
question : A quoi correspond `my.simple_page.uid` ? Est-ce qu'on a vraiment besoin de différencier ?

Reply:
Dans l'api Prismic, on fait une requête de la forme `my.{scustom-type}.{field}`, d'où le `my.simple_page.uid` pour récupérer le document.
cf: https://prismic.io/docs/javascript/query-the-api/query-predicates-reference